index.html

```diff
@@ -2,21 +2,25 @@
 <html lang="en">
 <head>
     <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;200&display=swap" rel="stylesheet"> 
+    <link rel="stylesheet" href="style.css">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Document</title>
     <script src="https://kit.fontawesome.com/d52fa491d2.js" crossorigin="anonymous"></script>
+    <title>Document</title>
 </head> 
 <body> 
     <header>
-        <nav> 
-            <a href="#">My Logo</a>
-            <i class="fa-solid fa-bars"></i>
-        </nav>
+        <ul>
+            <li><a href="#">My Logo</a></li>
+            <li><i class="fa-solid fa-bars"></i></li>   
+        </ul>
     </header>
     <section class="Headline"> 
         <h1>I’m kelvin Glad to see you!</h1>
-        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p id="one">I’m a software developer! I can help you build a product , feature or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+        <p id="two">LET'S CONNECT</p>
         <a href="#"></a>
         <div> 
```

Comment:
- [ ] You should wrap your social media icon in an `ul` tag and each one in a `<li>` tag and the social media should be using the  `<a>` tag  they should be clickable and take you to your social media accounts (if you have them)

- [ ] Also please make sure to export the icons from the Figma design and they should look the same as the one in the Figma design 
- Yours 
![image](https://user-images.githubusercontent.com/61790587/177768275-6e91bc45-30b7-44e5-9d24-51b77f13b4b0.png)
- Figma 
![image](https://user-images.githubusercontent.com/61790587/177768306-2d27c1a5-00fb-4b6d-b2d6-a1b04ebe633c.png)

Reply:
- As required by the previous reviewer, kindly do the following:
> * [ ]  Please make sure to export the icons from the Figma design and they should look the same as the one in the Figma design
> * [ ] Also, kindly ensure your icons have the correct spacing to match the design as much as you can..